business/istio_validations.go

```diff
@@ -553,18 +553,14 @@ func (in *IstioValidationsService) fetchNonLocalmTLSConfigs(mtlsDetails *kuberne
 		return
 	}
 
-	kubeCache := kialiCache.GetKubeCaches()[cluster]
-	if kubeCache == nil {
-		return
-	}
-
-	userClient := in.userClients[cluster]
-	if userClient == nil {
+	cfg := config.Get()
+	// TODO: Handle multi-primary instead of only using home cluster.
```

Comment:
Is there an issue for this?  We do claim MP support at this point.

Reply:
This is one of those things that should get cleaned up with moving things to the mesh page. I'll create a separate issue to capture that list.

Reply:
Created: https://github.com/kiali/kiali/issues/7203